search-index.json

```diff
@@ -0,0 +1,15 @@
+---
+layout: null
+---
+[
+{% for coll in site.collections %}
+{% for doc in coll.docs %}
+{
+  "id": {{ doc.url | jsonify }},
+  "title": {{ doc.title | jsonify }},
+  "text": {{ doc.content | strip_html | strip_newlines | jsonify }}
+},
+{% endfor %}
+{% endfor %}
+{"id":""} {% comment %} JSON arrays can't have trailing commas and using forloop.last won't work with nested loops so this is a quick hack {% endcomment %}
```

Comment:
Yes, this is ugly. Happy to take patches that remove the need for this, but I couldn't figure it out in 5 minutes so I did this instead.

Reply:
Haven't looked into it but could you do it with flatmap+forloop.last? If not, I'm okay with this hack.

Reply:
I couldn't find a flatmap back when I looked into this :(